### ISSUE TITLE
The app doesn't work on garena server

### ISSUE THREAD

**Issue by Nizaki** [molenzwiebel/Deceive#17]
it always show 
![image](https://user-images.githubusercontent.com/2827280/66625528-c0d6fe00-ec1e-11e9-9744-b1604dff3e7a.png)

this is leagueclient.exe that i selected
![image](https://user-images.githubusercontent.com/2827280/66625548-d3513780-ec1e-11e9-8b9e-3bf1ad389fa3.png)

**Comment by aPinat** [comment]
Yes, so Deceive actually doesn't work on Garena. Will try to explain what is happening and how to possibly fix it:

Here, Deceive erroring out on not finding LeagueClient.exe is because it checks for a "Logs" folder, beside the "Config" folder. But now even if you would add the "Logs" folder manually, Deceive would find and start the LeagueClient.exe correctly, but it still won't work on Garena servers, because you wouldn't be able to login. 
The auth system works differently than on Riot servers, since Garena uses it's own launcher. (Now the new Riot Client we got is actually kind of similar, but not quite the same...) When you start the game, the Garena launcher receives an auth token and passes it on to the LeagueClient as --garena-plus-token launch argument. This will log you into the client directly, bypassing the login screen with the login animations (what the Riot Client is doing now as well). Since Deceive can't just retrieve this token on its own with Garena servers, having Deceive launch League in Garena won't sign you in. 
Now I actually tried to write a program (initially to switch the language to English, when I was living in Thailand), that could get that token from Garena directly without Garena Plus and launch the client bypassing the bloated Garena client. That didn't work out as well, so I just decided to write a script that would just relaunch LeagueClient.exe in English after it had been started by Garena Plus.

So, a similar implementation would probably also work for Deceive, but I'd have to check that again once I get home, as I'm actually currently in Thailand as well ;-)
